RELEASE_NOTES.md

```diff
@@ -7,9 +7,15 @@ This file contains a high-level description of this package's evolution. Release
 
 ## 4.6.3 - TBD
 
-* Update the license from the home-brewed NetCDF license to the standard 3-Clause BSD License.  This change does not result in any new restrictions; it is merely the adoption of a standard, well-known and well-understood license in place of the historic NetCDF license written at Unidata.  This is part of a broader push by Unidata to adopt modern, standardized licensing.  
+## 4.6.2.1 - February 15, 2019
 
+### Maintenance Release
 
+* Update the license from the home-brewed NetCDF license to the standard 3-Clause BSD License.  This change does not result in any new restrictions; it is merely the adoption of a standard, well-known and well-understood license in place of the historic NetCDF license written at Unidata.  This is part of a broader push by Unidata to adopt modern, standardized licensing.  
```

Comment:
Should add:
* Now always write hidden coordinates attribute, which allows faster file opens when present (#1262).
* Some fixes for rename, including fix for renumbering of varids after a rename (#1307), renaming var to dim without coordinate var (#1297).
* Fix of NULL parameter causing segfaults in put_vars functions (#1265).
* Fix of --enable-benchmark benchmark tests (#1121).
* --disable-relax-coordinate-bound option deprecated, and will be removed in the next release. Use of this option can create builds that behave inconsistently on HPC systems. Coord bounds should always be relaxed, as is the current default. (#1010).